lints/0012_auth_allow_anonymous_sign_ins.sql

```diff
@@ -0,0 +1,56 @@
+create view lint."0012_auth_allow_anonymous_sign_ins" as
+
+with recursive role_members as (
+    select
+        roleid,
+        member
+    from pg_catalog.pg_auth_members
+    where roleid = (select oid from pg_roles where rolname = 'authenticated')
+    union
+    select
+        am.roleid,
+        am.member
+    from pg_catalog.pg_auth_members as am
+    inner join role_members as rm on am.roleid = rm.member
+),
+
+member_names as (
+    select r.rolname from pg_roles as r
+    inner join role_members as m on r.oid = m.member
+)
+
+select
+    'auth_allow_anonymous_sign_ins' as name,
+    'INFO' as level,
+    'EXTERNAL' as facing,
+    'Detects row level security (RLS) policies that allow access to anonymous users.' as description,
+    'https://supabase.github.io/splinter/0012_auth_allow_anonymous_sign_ins' as remediation,
```

Comment:
Could you change this to a main docs site link please? Following the pattern here: https://supabase.com/docs/guides/database/database-advisors